# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/fake/spicctrl/install-prefix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2004-11-21 21:49:22.881392144 +0100
+++ ./Makefile	2004-11-21 21:49:28.928472848 +0100
@@ -6,7 +6,7 @@
 LIBS =
 DOCS = AUTHORS CHANGES LICENSE
 VERSION = 1.9
-PREFIX = /usr/local
+PREFIX = /usr
 DESTBIN = $(PREFIX)/bin
 DESTMAN = $(PREFIX)/man/man1
 DESTDOC = $(PREFIX)/share/doc/spicctrl-$(VERSION)
